### DISCUSSION TITLE
What is the logic on Forecast from the FlowViz

### DISCUSSION THREAD

**Discussion by Karloyski** [nbrown02/FlowViz#108] [Q&A] [answered]
Hi Nick, I would like to know more about the forecast capability of the flowviz. Is there any other documentation related to this? what is the logic applied, what are the formulas being used in the calculations and the visuals. Thank you in advance!

**Comment by nbrown02** [comment, marked as answer]
Hey @Karloyski - the [Forecast page on the Wiki](https://github.com/nbrown02/FlowViz/wiki/3.-Forecast) provides further explanation. 

If you want to see the underlying javascript you can [read this](https://github.com/nbrown02/FlowViz/raw/main/Screenshots/WWIBD.docx) for _when will it be done_ or [this](https://github.com/nbrown02/FlowViz/raw/main/Screenshots/WWWG.docx) for _what will we get_

**Comment by Karloyski** [comment]
This is very helpful and again great Nick! much appreciated.

**Comment by nbrown02** [comment]
My pleasure :)
